bulletroboy/exoforce/exoforce.py

```diff
@@ -331,10 +339,13 @@ def __init__(self, cage_conf, node_name):
 		self.cage = Cage(cage_conf.cage_structure['height'], cage_conf.cage_structure['radius'], self.get_motors())
 
 		self.init_end_effectors()
+		self.init_force_decomp_params()
 
-		self.cage_state_publisher = self.create_publisher(CageState, '/roboy/simulation/cage_state', 1)
+		self.cage_state_publisher = self.create_publisher(CageState, Topics.CAGE_STATE, 1)
```

Comment:
you could also store the types of topics and do somth like 
```suggestion
		self.cage_state_publisher = self.create_publisher(Topics.CAGE_STATE_TYPE, Topics.CAGE_STATE, 1)
```

Reply:
yes, but we need to import those, so we will need also to store the package where the type is and call the import with exec()